commands/command_track.go

```diff
@@ -19,6 +19,8 @@ var (
 	}
 )
 
+// trackCommand takes a list of paths as an argument, and adds each path to the default
+// attribtues file (.gitattributes), if it's not already exist.
```

Comment:
This should be:
``// attributes file (.gitattributes), if it not already exists.``